plugins/pulp_rpm/plugins/importers/yum/existing.py

```diff
@@ -111,7 +111,7 @@ def check_all_and_associate(wanted, sync_conduit):
     for unit_type, values in sorted_units.iteritems():
         model = plugin_api.get_unit_model_by_id(unit_type)
         # FIXME "fields" does not get used, but it should
-        fields = model.unit_key_fields + ('_storage_path',)
+        # fields = model.unit_key_fields + ('_storage_path',)
```

Comment:
We will want to pass it into the controller method once we make that method support the extra parameter, so I think we should leave it and fix it ASAP.